Hai Reveal conversations: Rewrite A11 Sound The Alarm with fewer text walls and more player engagement

UnorderedSigh opened the pull request:
**Feature:** This PR rewrites the Hai Reveal [A11] Sound the Alarm mission to have more player engagement and shorter text.

Note: My work to reduce text walls has been split into smaller PRs by @MasterOfGrey's request, to make them easier to review. The changes to this mission were extensive enough that it needs its own PR.

## Feature Details
Conversations with especially long text are likely to be ignored, or at best skimmed, by players. That will make the Hai Reveal more confusing. Even worse, this mission's conversation is critical from a story standpoint, and that is lost on the player due to the wordy dialog. Unfortunately, I could not fix this by rearranging the text, so I had to rewrite most of the conversation.

These are the changes:

1. The Hai show Captain Last's attendance at the meeting is critical by speaking in Human Standard. This also increases player engagement.
2. They also ask the captain for advice, and the player is told this may be a "pivotal moment in history" so they're likely to read the text in its entirety.
3. The Elders make the same decision regardless of the players choices, letting the player share in Teeneep's frustration.
4. Teeneep gives a brief, but moving, speech that sounds like it was prepared in advance, instead of ranting.
5. The story links back to the activist organization by having her credit her supporters in the organization for the grassroots movement that led to her success. This supports her character as a uniter rather than a political mastermind or egotistical child.
6. Removed the option to eat with Alondo since it skipped critical information.
7. Sayari's warning to flee is put in its own scene (after a choice) so it is more noticable.
8. When an Elder that suggests 5000 kidnappings aren't important, some Elders are appalled.

## Testing Done
Replayed A11 many times.

[Friction Diction~3025-12-30 TRY7 before A11 with ships at darkcloak.txt](https://github.com/endless-sky/endless-sky/files/9996523/Friction.Diction.3025-12-30.TRY7.before.A11.with.ships.at.darkcloak.txt)

UnorderedSigh commented:
Pinging @bene-dictator again, hoping for a review.

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,43 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Teeneep explains her ominous words, "Our democracy allows a Petition of No Confidence against the government. If there are enough petitioners, it forces an election. Activists in my organization have been pushing a petition for weeks.`
+			`	"It was obvious what the Elder Council's response would be," and with a sarcastic smile, she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, I arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`
+			`	Sayari assures you, "These petitions take time, and rarely succeed. It is more of a warning to the Elders than anything else. Still, we should watch the situation.`
+			`	"For now," she says, "we're all in need of lunch"`
```
```suggestion
			`	"For now," she says, "we're all in need of some lunch."`
```

bene-dictator commented:
This looks a lot better and dramatic than the old one, it's a definite improvement.

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,43 @@ mission "Hai Reveal [A11] Sound the Alarm"
+				`	(Await the vote.)`
+			`	After a few more minutes of private discussions, the vote is tallied. Unfortunately, the majority vote "no" on both counts. Your words may not have swayed the vote, but perhaps they'll resonate in the Hai psyche long enough to have an effect.`
+			`	Teeneep is allowed to respond, and says ominously, "You know what will happen next."`
+			`	An angry Elder replies, "You wouldn't dare!" To which Teeneep responds, "It is already scheduled to happen unless I actively stop it!" She smiles and leaves the room as Sayari follows, motioning for you to do so as well.`
```
```suggestion
			`	An angry Elder replies, "You wouldn't dare!" to which Teeneep responds, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
```

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,43 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	Teeneep just shakes her head, pulls out a datapad, and quickly taps a few times. "Well, that was going to happen in an hour. No point waiting, though. Let's get dinner."`
+			`Upon your return, Teeneep sends an urgent demand for an audience with the Elders, along with a few paragraphs written in Hai. The meeting is called within an hour, and the room is entirely full. This may be the entire Elder Council; if so, they must consider this a grave matter. Teeneep and Sayari are waiting for you to follow them into the chambers.`
+			`	The Elder moderating the proceedings calls Teeneep to testify, using your language. It seems they consider your participation critical.`
+			`	Teeneep begins, "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. Will you do what is right for our people, or will you b ...
```
```suggestion
			`	Teeneep begins. "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. Will you do what is right for our people, or will you be replaced?"`
```

Teeneep is threatening the council here, but she doesn't say explicitly what the council should do. Is it better to leave her implying that the Hai military should accompany the Unfettered, or make her say it outright? She doesn't say anything about it here. I might have missed something from an earlier mission that covers this for the player, though.

UnorderedSigh replied:
The coordination with the Unfettered has to be much more than just a military escort. There's the matter of diplomacy with humans to prevent a war, and internal politics to get the Fettered Hai populace to accept it. I thought it would be best to avoid such a long discussion. Presumably, the council wouldn't get into such details unless they thought an agreement to work with Unfettered is likely. As such an agreement is almost unimaginable, I'd expect them to skip the talk unless the vote went in a deeply surprising direction.

UnorderedSigh replied:
I'm hoping more people will review, so I'll wait to hear more before I make such a large change to the conversation.

UnorderedSigh replied:
I remember now the original justification for the Unfettered help. The Fettered are beset on both sides by attackers, but Teeneep wants a deal where the Unfettered stop attacking the Fettered and guard the wormhole. 

That doesn't work anymore, now that the cease in intra-Hai hostilities is gone. I'll have to contrive another justification.

UnorderedSigh replied:
I think the best solution is for Teeneep to demand the Elders allow the Unfettered to pass through Fettered territory and assist in defending the wormhole. Ultimately, that is what ends up happening, but only a few Unfettered special ops ships rather than a full alliance.

UnorderedSigh replied:
Teeneep now insists on an agreement with the Unfettered that is also negotiated with the humans. That way, the player's continued cooperation isn't suspicious (as they may be intentionally starting a war otherwise). I've updated the player's possible responses to match the new Teeneep speech.

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,43 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	The Elder moderating the proceedings calls Teeneep to testify, using your language. It seems they consider your participation critical.`
+			`	Teeneep begins, "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. Will you do what is right for our people, or will you b ...
+			`	At the word "replaced," murmurs rise and Sayari looks at you with deep concern. The moderator silences the room and asks you to testify. "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. What do you advise?`
+			`	You carefully choose your words before speaking, as this may be a pivotal moment in history. This may spark a war - or prevent one.`
```
```suggestion
			`	You should carefully choose your words before speaking, as this may be a pivotal moment in history. This may spark a war - or prevent one.`
```

UnorderedSigh commented:
This mission conversation touches on Fettered-Unfettered interaction, so I'm hoping for a review from @Hurleveur and @EjoThims

Hurleveur commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,43 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			action
+				set "told hai to reject unfettered help"
+			`	A disgruntled Elder responds, "Change our decision? Our decision not to violate human sovereignty? Even if I lose my Eldership, I will not vote for violence."`
+			`	Teeneep says, "If you take no action, and refuse help from our brethren, they'll pass through our territory anyway. Fighting at Wah Ki has stopped, as the military is anticipating an agreement."`
```
I think that if fighting stopped at wah ki we need to make it actually stop trough an event

UnorderedSigh replied:
Can you please find where the code is that stops the fighting?

I don't know where it happens, but it isn't in this mission. It's earlier.

Hurleveur replied:
Oh so it actually happens? Because the old text said it had intensified or smth?

UnorderedSigh replied:
It is possible I removed it by accident. I'll replay the Hai Reveal and make sure it's still there.

Hurleveur replied:
I played trough HR I'm pretty sure it did not happen

UnorderedSigh replied:
It definitely should happen, as the rest of Hai Reveal doesn't make sense without it.

UnorderedSigh replied:
I don't want to touch that in this PR though. Filling that plot hole would require a code change in the Wanderers campaign too. Instead, I'll just remove Teeneep's statement.

Hurleveur replied:
Idk if it would require changes in Wand honestly
Yeah also they say it intensifies since the start but it didnt change one bit bruh

Could be for another PR for sure

UnorderedSigh replied:
The change in the Wanderers campaign is to restore the hostility. In other words, the Hai Reveal would change the affinity so the Hai factions are friends, but Sayari's visit will change them back to enemies.

That definitely does *not* belong in this PR though, so I'll just modify the dialog so it doesn't need a new government affinity.

UnorderedSigh replied:
After re-reading the Hai Reveal/Wanderers campaign connections, I see some bigger plot holes. Somehow the enormous political clout Teeneep had mysteriously never existed in the Wanderers campaign. She also seems to be hugely pro-Unfettered, not caring much about the Wanderers, even though she put a lot of work into keeping peace with the humans during their dealings with the Unfettered. This is easy to fix without substantive changes to the Wanderers campaign, but I'll work on that after 7586 has more review.

Hurleveur replied:
> The change in the Wanderers campaign is to restore the hostility. In other words, the Hai Reveal would change the affinity so the Hai factions are friends, but Sayari's visit will change them back to enemies.
> 
> That definitely does _not_ belong in this PR though, so I'll just modify the dialog so it doesn't need a new government affinity.

Oh it makes them friendly I thought it stopped the incoming ships
I saw a new gov "friendly uhai" was made so I thought that wasnt the case

UnorderedSigh replied:
No, that's only for the ships the Hai allow to pass through their territory. I'm 83% certain all other Unfettered fleets are in the original government.

UnorderedSigh replied:
Teeneeps prior mention of Unfettered not attacking is now replaced with a warning: the Fettered probably could not stop the entire Unfettered armada, and it is unnecessary bloodshed anyway.

Hurleveur commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,43 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Sayari assures you, "These petitions take time, and rarely succeed. It is more of a warning to the Elders than anything else. Still, we should watch the situation.`
+			`	"For now," she says, "we're all in need of some lunch."`
+			choice
+				`	(Go to lunch.)`
```
just wondering why did you remove the option to have lunch with diplomats?

UnorderedSigh replied:
It causes you to miss critical plot points.

UnorderedSigh replied:
That's in the PR description, "Removed the option to eat with Alondo since it skipped critical information."

Hurleveur commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,43 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	She scrolls down on her pad several times and begins to mutter, "Oh dear, oh no." She lapses into Hai for a moment before returning. "It hit the trending issues board, and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action and it's only been two hours... this is what you hope ...
-			`Teeneep places a gentle hand on Sayari's arm. "Better this than war."`
-			label alarm
+				`	"You can trust the Unfettered to defend you, but you must be careful to avoid a war with the humans. That would be catastrophic for everyone."`
```
I really like that they're asking your opinion for the UHai
Mb they could ask for the kidnapping threat as well? Since you're most expert in that, and actually saw victims first-hand

Idk why FL would be an UHai expert more than they are honestly, but kidnapping is where the expertise is at (though doesnt hurt to ask for UHai sincerity)

UnorderedSigh replied:
I meant for the moderator's question to imply that he's asking about the full situation, including the kidnappings and the tensions with human governments. Perhaps I need to carefully reword his question.

Hurleveur replied:
I just feel like thats where they should ask your opinion mostly

UnorderedSigh replied:
Yes, that makes sense. I'll have them ask that before they ask about the Unfettered decision.

UnorderedSigh replied:
I added a question to the player about the kidnappings.

Hurleveur reviewed (commented):
Don't worry I don't want all uhai to pass by me; that said I'm always willing to review things touching them so ty for notifying me
Anyway here are some thoughts about it, although I'm no pro on content ofc

Hurleveur reviewed (approved):
looks good to me, do note I'm no content reviewer though

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,54 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	She scrolls down on her pad several times and begins to mutter, "Oh dear, oh no." She lapses into Hai for a moment before returning. "It hit the trending issues board, and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action and it's only been two hours... this is what you hope ...
-			`Teeneep places a gentle hand on Sayari's arm. "Better this than war."`
-			label alarm
+				`	"There evidence is circumstantial put compelling, and we must expand the investigation."`
```
```suggestion
				`	"There evidence is circumstantial but compelling. We must expand the investigation."`
```

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,54 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			label uncertainties
+			`	Teeneep looks mildly perturbed, while many Elders nod with agreement.`
+			label "next question"
+			`	The moderator looks deeply serious, says, "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. We have become unwittingly embroiled in human internal conflicts, and attacked for the first time in 300 years. Much of this is beyond our experience, but perhaps not beyond yours. What do you advise?"`
```
```suggestion
			`	The moderator looks deeply serious. He says, "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. We have become unwittingly embroiled in human internal conflicts, and have been attacked for the first time in 300 years. Much of this is beyond our experience, but perhaps not beyond yours. What do you advise?"`
```

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,54 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Teeneep is allowed to respond, and says ominously, "You know what will happen next."`
+			`	An angry Elder replies, "You wouldn't dare!" to which Teeneep responds, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
+			`	Sayari leads you to a private meeting room.`
+			`	Teeneep explains her ominous words, "Our democracy allows a Petition of No Confidence against the government. If there are enough petitioners, it forces an election. Activists in my organization have been pushing a petition for weeks.`
```
```suggestion
			`	Teeneep explains her ominous words. "Our democracy allows a Petition of No Confidence against the Elders. If there are enough petitioners, it forces an election. Activists in my organization have been pushing a petition for weeks.`
```

Hurleveur commented:
There is smth missing in HR I think, from the uhai side.
With the option of a great crusade against the humans even being as unlikely as it could be, the Uhai should do everything to have that happen, instead of just dealing with the smaller picture of kidnapping. This does give then popularity but means missing the bigger picture here.

I'd expect them to try and turn all hai against the humans for the kidnappings alone, not help in a special ops to take them down...

That said, I think both options could coexist depending on what you do (or do not do) to show them humans can be trusted. The warlord could be honorable with the uhai elders being less so..

Considerations prob partly best left for another PR though

UnorderedSigh commented:
The Unfettered see Teeneep as a possible route to peaceful unification, which would give them access to the vast resources of the Fettered without a fight. It would also be hard to maintain any gains in human space without the assistance of the Fettered. I think the Unfettered leaders are wise enough strategists to seek long-term gains. Actually, the whole story revolves around the Unfettered leaders being wise enough for long-term planning.

Hurleveur commented:
I never meant them fighting the humans alone, ofc.
I meant them trying to light the flame of the hai-human conflict and help turning the tide significantly in their favor, with the war being the catalyst to their unification.

UnorderedSigh commented:
> I meant them trying to light the flame of the hai-human conflict and help turning the tide significantly in their favor, with the war being the catalyst to their unification.

I expect we'll need a lot of changes in Hai Reveal to connect it properly to the Unfettered campaign. I've put links in the dialog where the player could take things in that direction. The Hai Reveal in its current state deliberately does not affect the possibility of war since we don't know how that will play with later campaigns. Instead, all it does is add more possibilities for directions Fettered-Unfettered-Human relations can go.

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,54 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	She scrolls down on her pad several times and begins to mutter, "Oh dear, oh no." She lapses into Hai for a moment before returning. "It hit the trending issues board, and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action and it's only been two hours... this is what you hope ...
-			`Teeneep places a gentle hand on Sayari's arm. "Better this than war."`
-			label alarm
+				`	"The evidence that they are kidnappings is circumstantial but compelling. We must expand the investigation."`
```
```suggestion
				`	"The evidence that there are ongoing kidnappings is circumstantial, but compelling. We must expand the investigation."`
```
Whoops, messed this one up last time.

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,54 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	An angry Elder replies, "You wouldn't dare!" to which Teeneep responds, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
+			`	Sayari leads you to a private meeting room.`
+			`	Teeneep explains her ominous words. "Our democracy allows a Petition of No Confidence against the Elders. If there are enough petitioners, it forces an election. Activists in my organization have been pushing a petition for weeks.`
+			`	"It was obvious what the Elder Council's response would be," and with a sarcastic smile, she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, I arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`
```
```suggestion
			`	"It was obvious what the Elder Council's response would be," and with a sarcastic smile she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, I arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`
```
I'd like to try get rid of this repetition with the smiling.

UnorderedSigh replied:
I changed it to "with a bit of sarcasm"

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,28 +1141,54 @@ mission "Hai Reveal [A11] Sound the Alarm"
+				`	(Await the vote.)`
+			`	After a few more minutes of private discussions, the vote is tallied. Unfortunately, the majority vote "no" on both counts. Your words may not have swayed the vote, but perhaps they'll resonate in the Hai psyche long enough to have an effect.`
+			`	Teeneep is allowed to respond, and says ominously, "You know what will happen next."`
+			`	An angry Elder replies, "You wouldn't dare!" to which Teeneep responds, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
```
```suggestion
			`	An angry Elder replies, "You wouldn't dare!" to which Teeneep yells back, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
```
Now I think 'responds' is a little dull, so I'm going to change it again to 'yells back'.

UnorderedSigh commented:
I made a key change just now, so Teeneep's character in this story is more consistent with her portrayal earlier in part 4. Instead of bragging about her success, Teeneep credits her supporters. This is more in line with her abilities as a uniter, rather than a political mastermind.

UnorderedSigh commented:
There's a detail I forgot to mention. @NomadicVolcano told me that workings of the Elder Council and Hai government are supposed to be as vague as possible, since the Hai are supposed to have a stable form of government beyond what we can understand today. Since we don't understand it, we have to leave it vague.

I've made the functions of the Elder Council as vague I can while still letting the player meaningfully affect decisions.

I leave these as open questions to maintain the mystery of the Elder Council:

1. Why do they sometimes vote, and sometimes seem to already know the answer?
2. Who is this "moderator" that is leading the discussion, what powers do they have? Is this a leader? A minor functionary speaking prepared lines? One of the Elders?
3. Why do some Elders have to be called to speak, but some are allowed to speak out of turn? Is this a difference in status or function of the Elder?
4. Teeneep provided a few paragraphs, and then everyone seemed to know what questions were to be asked. Was there some kind of behind-the-scenes discussion? Is this not the real Elder Council meeting, but a formality to announce decisions? Or are they really that stable and united that they don't need extensive discussions?
5. Why are there sometimes three Elders, sometimes a handful, and sometimes a room full of them?

UnorderedSigh commented:
> workings of the Elder Council and Hai government are supposed to be as vague as possible

I have been informed that NV's assertion may be complete nonsense, and I should ask MasterOfGrey.

Hurleveur commented:
I would love those to be true.
Currently this looks too similar to what the Wanderers have and what we have today, it should leave the player with a lot of questions, not be familiar. The Unfettered can have a simpler system for instance, in contrast, but not the Hai

UnorderedSigh commented:
Yes, we need the right amount of mystery while still having enough consistency for the player to have meaningful conversations. I hope I achieved that, but I may have only achieved player confusion.

UnorderedSigh commented:
> Currently this looks too similar to what the Wanderers have and what we have today

I think that's because we haven't had any major government-specific stories outside human space. Even the Republic political system is defined only at a vague level, no more than is needed for the plot. I think the Hai need their own vague feel, that is alien but familiar enough for interaction.

UnorderedSigh commented:
What I'm trying to say is that the Wanderers seem to have a "you meet some people, and they decide something" decision-making system, because the plot doesn't need more detail. The Hai have that as well, outside this PR. The player is manipulating Hai politics, so I need a bit more detail to have an interactive conversation between the player and various meetings of Elders. Not so much detail as to make it feel familiar, though.

UnorderedSigh commented:
I just removed all action choices, which required reworking some sections. Also, I moved Sayari's panicked warning to flee into its own scene, so it is more noticeable.

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,29 +1141,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Another Elder interjects, "You're exaggerating the impact, Teeneep. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone."`
+			`	Many Elders seem appalled by that observation, but the moderator interrupts. "We will vote on two counts: Do we act? Do we accept Unfettered help?"`
+			choice
+				`	"Teeneep, what happens next?"`
```
```suggestion
				`	"What's going to happen, Teeneep?"`
```

EjoThims reviewed (changes requested):
Almost exclusively commas, one way or the other.

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,29 +1141,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			label uncertainties
+			`	Teeneep looks mildly perturbed, while many Elders nod with agreement.`
+			label "next question"
+			`	The moderator turns back to you with a deeply serious face. "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. We have become unwittingly embroiled in human internal conflicts and have been attacked for the first time in 300 years. Much of this is beyond our experience, but perhaps not beyond yours. What do you advise?"`
```
```suggestion
			`	The moderator turns back to you with a highly serious face. "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. We have become unwittingly embroiled in human internal conflicts and have been attacked for the first time in 300 years. Much of this is beyond our experience, but perhaps not beyond yours. What do you advise?"`
```
Get rid of some repetition that could happen.

bene-dictator commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1141,29 +1141,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			label frantic
+			`	Not half way through her food, Sayari's data pad beeps frantically. She looks at it and mutters, "Oh dear, oh no." She lapses into Hai for a moment before returning. "The petition! It hit the trending issues board and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action, and it ...
+			`	Teeneep places a gentle hand on Sayari's arm. "Better this than war."`
+			`	Sayari's eyes are alive with thinly-veiled panic as she slams her hand down on the table. Teeneep seems shocked, as if she had never seen Sayari so emotional.`
```
```suggestion
			`	Sayari's eyes are alive with thinly-veiled panic as she slams her paw down on the table. Teeneep seems shocked, as if she had never seen Sayari so emotional.`
```
I don't like using paw instead of hand, but it's more consistent with everything else.

MasterOfGrey commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1233,29 +1233,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	Sayari's eyes are alive with thinly-veiled panic as she slams her hand down on the table. "Captain <last>, you must leave immediately. Get on your ship and fly to <planet> right now! The winds of change are afoot and we will have an election. They'll hit the threshold in maybe only twenty minutes, and it'll take maybe thirty more for the bureaucracy to act on it. The instant the election is  ...
+			`Upon your return, Teeneep sends an urgent demand for an audience with the Elders, along with a few paragraphs written in Hai. The meeting is called within an hour, and the room is entirely full. This may be the entire Elder Council; if so, they must consider this a grave matter. Teeneep and Sayari are waiting for you to follow them into the chambers.`
+			`	The Elder moderating the proceedings calls Teeneep to testify, using your language. It seems they consider your participation critical.`
+			`	Teeneep begins. "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. They are ready to send a fleet to human space and extin ...
```
I went to make three different suggestions but none of them were right.

This opening sentence has the following problems:

- It was less than 300 years (also we're trying not to be too specific about that because the date the player gets to this varies).
- If this issue is ongoing/cyclical then it's not the first time, and describing it as such weakens her position.
- We have not necessarily established why the council is inactive, or at least not presented the possible motivations (I don't think - feel free to correct me) which makes this accusation incredibly rude.
- Also, it's bad politic to directly insult the people you wish to galvanise to action when you've already indirectly insulted them by working behind their backs. This current presentation implies she *wants* them to refuse her...
- The Unfettered never stopped wanting to be defenders of all Hai, the only thing they've been convinced of is a willingness to properly fight the Hai over the moral issue of inaction.
- Also, most Unfettered alive were probably not in fact the actual defenders. They'd be a minority at this stage.
- Extinguishing the pirate threat entirely is not really possible without mass conducting atrocities, which the Elders know.
- What they've already been doing could well be described as "negotiating with humans", what's being asked is for them to demand action from specific humans (The Republic).

These Elders have been politicking for a long time. I wouldn't imagine that Teeneep would get to the end of this paragraph successfully without being interrupted a half dozen times and likely dismissed by half those assembled.

MasterOfGrey commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1233,29 +1233,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	At the word "replaced," murmurs rise, and Sayari looks at you with deep concern. The moderator silences the room and asks you to testify.`
+			`	"Captain <last>, can you speak to the truth of these disappearances?"`
+			choice
+				`	"The evidence that there are ongoing kidnappings is circumstantial, but compelling. We must expand the investigation."`
+					goto uncertainties
+				`	"It is a coordinated kidnapping campaign, and we must put a stop to it but take care not to start a war."`
```
These are both comparatively a lot of words for <first> to say in a choice.
They should be re-assessed when the preceding text changes.

MasterOfGrey commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1233,29 +1233,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			label uncertainties
+			`	Teeneep looks mildly perturbed, while many Elders nod with agreement.`
+			label "next question"
+			`	The moderator turns back to you with a highly serious face. "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. We have become unwittingly embroiled in human internal conflicts and have been attacked for the first time in 300 years. Much of this is beyond our experience, but perhaps not beyond yours. What do you advise?"`
```
Again, the 300 years.
However, in this context it is worth considering that the player is not a politician, their character has no diplomatic or policy backgrounding.
If some subset of the Elders are actually engaged enough to want to address possible options here, it seems to me far more likely that they would speak of a selection of options that they've already been given or requested from various people (or have devised themselves) and query you on which one you think is most reasonable/problematic. (As an outsider you can't be expected to propose a solution, but *can* be expected to spot a problem that others might have overlooked.)

I would expect that the options they'd present would looks something like:
- Their favourite
- The one they think is most practical
- The one they think is most ridiculous

Not necessarily in that order, and not necessarily with any indication of which is which.

MasterOfGrey commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1233,29 +1233,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Teeneep responds, "They'll vote to do nothing. It's all this council knows how to do." Sayari sighs loudly at this but doesn't contradict Teeneep.`
+			label results
+			`	After a few more minutes of private discussions, the Elders vote "no" on both counts. Your words may not have swayed the vote, but perhaps they'll resonate in the Hai psyche long enough to have an effect.`
+			`	Teeneep is allowed to respond and says ominously, "You know what will happen next."`
+			`	An angry Elder replies, "You wouldn't dare!" Teeneep yells back, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
+			`	Sayari leads you to a private meeting room.`
```
This sequence is gratuitous and feels forced now.

UnorderedSigh replied:
I left this awkward part in because an Elder is supposed to threaten Teeneep. I will come up with a different way for the Elder to threaten.

MasterOfGrey commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1233,29 +1233,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Teeneep is allowed to respond and says ominously, "You know what will happen next."`
+			`	An angry Elder replies, "You wouldn't dare!" Teeneep yells back, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
+			`	Sayari leads you to a private meeting room.`
+			`	Teeneep explains her ominous words. "Our democracy allows a Petition of No Confidence against the Elders. If there are enough petitioners, it forces an election. Activists in the organization I founded have led a grassroots movement pushing a petition for weeks.`
+			`	"It was obvious what the Elder Council's response would be," and with a bit of sarcasm she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, my supporters arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`
+			`	Sayari assures you, "These petitions take time and rarely succeed. It is more of a warning to the Elders than anything else. Still, we should watch the situation.`
```
One of the objectives of the original piece was to show the Hai as a politically engaged people, but a people that are undecided.
A people equipped with a mechanism for making direct and radical change, but a predilection for disagreeing on thing sufficiently that it rarely amounted to being more than an opinion poll for the Elder's to observe the results of.

This also has issues with being presented as a grassroots movement:

- Such movements tend to take more than a few weeks.
- The Hai population is not small, to push a movement like that quickly would require Teeneep's organisation to be *very* large. (Which is why originally she was leveraging existing infrastructure, with the aid of her organisation's influence.)
- You know the whole bit about needing to confront the Elders directly and conform to the process? That is *directly* at odds with the concept of making sensitive information publicly available in the background as part of some social campaign.
- Corollary: if that campaign were successful enough to already be at this stage, the Elders would have heard about it already and she'd have been summarily censured for breach of process and betrayal of trust.

MasterOfGrey commented on data/hai/hai reveal 4 middle.txt:
```diff
@@ -1233,29 +1233,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			label frantic
+			`	Not half way through her food, Sayari's data pad beeps frantically. She looks at it and mutters, "Oh dear, oh no." She lapses into Hai for a moment before returning. "The petition! It hit the trending issues board and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action, and it ...
+			`	Teeneep places a gentle hand on Sayari's arm. "Better this than war."`
+			`	Sayari's eyes are alive with thinly-veiled panic as she slams her paw down on the table. Teeneep seems shocked, as if she had never seen Sayari so emotional.`
+			choice
+				`	"What's wrong, Sayari?"`
+			`	She says frantically, "Captain <last>, you must leave immediately!`
```
Everything about this sequence felt correct but off in how it was presented.
```suggestion
			`	Sayari pauses; staring at her friend for a long moment before breaking contact. She draws a surprised gasp from Teeneep as she suddenly slams her paw down on the table and turns to you, her eyes wide with thinly-veiled panic.`
			`	"<last>, you must leave immediately!" she says, frantically.`
			choice
				`	"What's wrong?"`
```

MasterOfGrey reviewed (changes requested):
This is one case where the goal was good, but some of the efforts have introduced more problems than solutions.

There are some things where, if we're going to be specific about what was or wasn't said, then we need to be much better at it.

UnorderedSigh commented:
I've discussed with MasterOfGrey how to fix this PR, and we have a specific plan. It'll take me a few days to implement it.
